src/classes/issues-processor.ts

```diff
@@ -206,6 +206,19 @@ export class IssuesProcessor {
         continue; // don't process exempt issues
       }
 
+      const anyOfLabels: string[] = wordsToList(this.options.anyOfLabels);
+      if (
+        anyOfLabels.length &&
+        !anyOfLabels.some((label: Readonly<string>): boolean =>
+          isLabeled(issue, label)
+        )
+      ) {
```

Comment:
Some is only called if the array is not empty.
So the check regarding the array length seems useless to me.
When the array is empty, it will return `false` which is exactly what you can expect.
```suggestion
      if (!anyOfLabels.some((label: Readonly<string>): boolean =>
          isLabeled(issue, label)
        )
      ) {
```
As a side-note, I think that having a private method would make this easier to read.

Reply:
@C0ZEN Thanks for the quick review! Well, you're correct, `.some()` does return `false` on an empty array, however in this case we do not want to execute the code inside the `if` block in that case (note the `!`). If we remove the check for empty array then we'd be skipping all issues.

Reply:
Yep sorry, the logic is reversed here because we need to skip if we have some required label and if the current processed issue don't have one of these.
In that case, `includes` make it a little bit more cleaner IMO.
```suggestion
      if (anyOfLabels.length && !anyOfLabels.includes(isLabeled(issue, label))) {
```

Reply:
@C0ZEN But `.includes()` would be expecting a `string` in this case, and as far as I can tell `isLabeled()` returns a `boolean`.

Reply:
Yeah you are right I was thinking of having a fn to get the issue label but it's more work for only a bit of sugar so better stick with this, sorry for the bad suggestions I think it's time for me to hit the bed 😂